pallets/tips/src/tests.rs

```diff
@@ -157,6 +159,36 @@ impl pallet_treasury::Config for Test {
     type MaxApprovals = MaxApprovals;
     type ProposalBondMaximum = ();
 }
+
+const MILLICENTS: Balance = 10_000_000_000_000;
+const CENTS: Balance = 1_000 * MILLICENTS;
+const DOLLARS: Balance = 100 * CENTS;
+
+parameter_types! {
+    pub const ClassDeposit: Balance = 100 * DOLLARS;
+    pub const InstanceDeposit: Balance = 1 * DOLLARS;
+    pub const KeyLimit: u32 = 32;
+    pub const ValueLimit: u32 = 256;
+    pub const StringLimit: u32 = 50;
+}
+
+impl pallet_uniques::Config for Test {
+    type Event = Event;
+    type ClassId = u32;
+    type InstanceId = u32;
+    type Currency = Balances;
+    type ForceOrigin = frame_system::EnsureRoot<u128>;
+    type ClassDeposit = ConstU64<2>;
+    type InstanceDeposit = ConstU64<1>;
+    type MetadataDepositBase = ConstU64<1>;
+    type AttributeDepositBase = ConstU64<1>;
+    type DepositPerByte = ConstU64<1>;
+    type StringLimit = ConstU32<50>;
+    type KeyLimit = ConstU32<50>;
```

Comment:
These 3 `limit` values were set to 50, is it enough? can we change it in the future?

Reply:
in runtime/src/lib.rs
```
    pub const KeyLimit: u32 = 32;
    pub const ValueLimit: u32 = 256;
    pub const StringLimit: u32 = 50;
```
    KeyLimit and ValueLimit would be enough, StringLimit change to 256 would be fine.